### ISSUE TITLE
[PhpUnitBridge] Shebang is printed in the output while using bin/phpunit

### ISSUE THREAD

**Issue by kiler129** [symfony/symfony#27035]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| Symfony version  | 4.1

Currently while running `bin/phpunit` as Flex recommends the first line of output contains shebang line. This makes usage of `bin/phpunit` impossible with e.g. PhpStorm, since version output looks like that:

```
$ ./bin/phpunit --version
#!/usr/bin/env php
PHPUnit 6.5.8 by Sebastian Bergmann and contributors.

$ 
```

The problem arises from the fact the generated `bin/phpunit` which is an executable with shebang contains require for `simple-phpunit` which also contains shebang. This may be non-bug if usage of `./bin/phpunit` is discouraged (https://github.com/symfony/symfony-docs/issues/9679, but why is it still generated?)

**Comment by nicolas-grekas** [comment]
That's true, but there is nothing we can do about it AFAIK...

**Comment by kiler129** [comment]
I was fiddling with this on Friday and I edited `bin/phpunit` to use [`passthru()`](http://php.net/manual/en/function.passthru.php) instead of including `simple-phpunit`. It spawns another process, but it works as expected.

I honestly see two solutions here:
1. Use `passthru()` while generating `bin/phpunit` to execute `simple-phpunit`
2. Extract the actual code from `simple-phpunit` to a separate file and include that file in both `simple-phpunit` and `bin/phpunit`

From my perspective second solution is better, since we're not creating another process.

**Comment by nicolas-grekas** [comment]
passthru() was used before but this has been changed for inline, which provides several benefits (eg allowing php -d...)
I don't know about 2., PR always welcomed :)

**Comment by nicolas-grekas** [comment]
Closing as "acceptable glitch - additional complexity unwanted" :)

**Comment by JustBlackBird** [comment]
@nicolas-grekas I've implemented the second solution :)

**Comment by fabpot** [comment]
fixed in #31364
